packages/story-editor/src/karma/fixture/db/getTaxonomiesResponse.js

```diff
@@ -15,9 +15,79 @@
  */
 
 export default [
+  {
```

Comment:
move categories to top of array because this is the only reason that that sort of the taxonomies array was needed

Reply:
💭 I initially added the sorting because the api response while running it locally had categories underneath tags. If that's changed then we don't need this

Reply:
We should not rely on the amount or order of taxonomies in the API response, nor should we care.

There might be the taxonomies `categories, tags`, or there might be the taxonomies `colors, tags, categories, foo, bar`.

The taxonomy panel needs to be built in an agnostic way and not artificially sort or hide taxonomies or anything.